Gems/ROS2/Code/Source/RobotImporter/Pages/PrefabMakerPage.cpp

```diff
@@ -34,9 +34,11 @@ namespace ROS2
         m_spawnPointsComboBox = new QComboBox(this);
         m_spawnPointsInfos = allActiveSpawnPoints.values;
 
-        for (int i = 0; i < allActiveSpawnPoints.values.size(); i++)
+        m_spawnPointsComboBox->addItem("default", QVariant(0));
```

Comment:
Move "default" to constexpr and reuse it in isDefault method.
Do not forget about i18n and _tr("") macro in Qt code.